### ISSUE TITLE
Memory usuage during validation

### ISSUE THREAD

**Issue by 1980x** [TXH-mercury/VAST#9]
Hi, When validation set size is large, GPU memory usage is much more than required during training even with small batch size. Can you please suggest where is the issue?
Thanks
